assets/js/modules/analytics/components/module/ModulePopularPagesWidgetGA4.js

```diff
@@ -231,20 +239,27 @@ function ModulePopularPagesWidgetGA4( props ) {
 		},
 	];
 
-	const rows = report?.rows?.length ? cloneDeep( report.rows ) : [];
-	// Combine the titles from the pageTitles with the rows from the metrics report.
-	rows.forEach( ( row ) => {
-		const url = row.dimensionValues[ 0 ].value;
-		row.dimensionValues.unshift( { value: titles[ url ] } ); // We always have an entry for titles[url].
-	} );
+	let rows = report?.rows?.length ? cloneDeep( report.rows ) : [];
+	let ZeroState = ZeroDataMessage;
+	// Use a custom zero state when the GA4 reporting tour is active
+	if ( isGA4ReportingTourActive ) {
+		rows = [];
+		ZeroState = () => <PreviewTable rows={ rows.length || 10 } />;
+	} else {
+		// Combine the titles from the pageTitles with the rows from the metrics report.
+		rows.forEach( ( row ) => {
+			const url = row.dimensionValues[ 0 ].value;
+			row.dimensionValues.unshift( { value: titles[ url ] } ); // We always have an entry for titles[url].
+		} );
```

Comment:
This is the only part which really relies on things being loaded so we only run it here to avoid the errors we would otherwise get.